docs/faq.rst

```diff
@@ -1118,6 +1118,21 @@ checking for free disk space::
 
     borg config -- $REPO_LOCATION additional_free_space -2T
 
+How do I rename a repository?
+-----------------------------
+
+There is nothing special that needs to be done, you can simply rename the
+directory that corresponds to the repository. However, the next time borg
+interacts with the repository (i.e, via ``borg list``), depending on the value
+of ``BORG_RELOCATED_REPO_ACCESS_IS_OK``, borg may warn you that the repostiory
+has been moved. You will be given a prompt that confirms you are OK with this.
+
+If ``BORG_RELOCATED_REPO_ACCESS_IS_OK`` is unset, borg will interactively ask for
+each repository whether it's OK.
+
+If you are renaming multiple repositories, it may be useful to set
+``BORG_RELOCATED_REPO_ACCESS_IS_OK=yes`` to avoid the prompts.
```

Comment:
Multiple might be annoying, but the more fundamental thing is interactive querying vs. non-interactive scripting and assuming it is ok.